specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/operations.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
Does it mean we need to add the operations file for any new version? it doesn't matter waht is the change?

Reply:
unfortunately yes